commandapi-platforms/commandapi-paper/commandapi-paper-core/src/main/java/dev/jorel/commandapi/arguments/AsyncPlayerProfileArgument.java

```diff
@@ -1,34 +1,37 @@
 package dev.jorel.commandapi.arguments;
 
+import com.destroystokyo.paper.profile.PlayerProfile;
 import com.mojang.brigadier.context.CommandContext;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
 import dev.jorel.commandapi.CommandAPIBukkit;
+import dev.jorel.commandapi.CommandAPIPaper;
 import dev.jorel.commandapi.executors.CommandArguments;
 import org.bukkit.OfflinePlayer;
 
+import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
 /**
  * An argument that represents the Bukkit OfflinePlayer object with asynchronous support.
  *
  * @since 9.7.1
  */
-public class AsyncOfflinePlayerArgument extends SafeOverrideableArgument<CompletableFuture<OfflinePlayer>, OfflinePlayer> {
+public class AsyncPlayerProfileArgument extends SafeOverrideableArgument<CompletableFuture<List<PlayerProfile>>, PlayerProfile> {
```

Comment:
```suggestion
public class AsyncPlayerProfileArgument extends SafeOverrideableArgument<CompletableFuture<PlayerProfile>, PlayerProfile> {
```

Should we return a list of `PlayerProfiles`? If this always resolves to a list with a single element, we should probably just return that element. Or does this actually give multiple players?

Reply:
It can return multiple profiles when using selectors.

Reply:
Ah, so I suppose these javadocs should be updated instead

https://github.com/CommandAPI/CommandAPI/blob/63d55a5d587773ff947fdae56a0a6ef88b62e5f8/commandapi-platforms/commandapi-paper/commandapi-paper-core/src/main/java/dev/jorel/commandapi/arguments/AsyncPlayerProfileArgument.java#L21-L26